src/styles/cardfolder/bottombutton.scss

```
@@ -0,0 +1,23 @@
+.card__purchase-button {
+  margin-top: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  height: 40px;
+  width: 166px;
+  background-color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.card__purchase-button:hover {
+  background-color: #fff;
+  color: #00acdc;
+}
```

Use scss nesting, fix it everywhere
```suggestion
  text-transform: uppercase;
  color: #fff;
  
  &: {
    // write hover styles here
  }
}

```